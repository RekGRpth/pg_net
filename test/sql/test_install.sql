create extension curl_worker;

select http.async_get('http://supabase.io');

select *
from http.request_queue;

