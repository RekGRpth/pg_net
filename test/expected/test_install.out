create extension curl_worker;
select http.async_get('http://supabase.io');
 async_get 
-----------
         1
(1 row)

select *
from http.request_queue;
 id |        url         | params | headers | timeout_seconds | curl_opts 
----+--------------------+--------+---------+-----------------+-----------
  1 | http://supabase.io | {}     | {}      |               1 | {}
(1 row)

